src/environments/environment.local.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const environment = {
```

Comment:
This will need reworked given the changes we made to allow runtime config. Take a gander here:

https://github.com/sonatype-nexus-community/search-maven-org/tree/master/src/assets/data

And here:

https://github.com/sonatype/search-maven-org/tree/master/src/app/shared/config

And here:

https://github.com/sonatype/search-maven-org/blob/master/build.sh

The last one I wasn't super stoked on, but Angular seems to not handle fileReplacement very well with regards to the json files, so I turned to a build.sh script.

Reply:
Yep, agreed. Will do.